crates/bevy_window/src/window.rs

```diff
@@ -126,7 +126,7 @@ impl NormalizedWindowRef {
 )]
 #[reflect(Component, Default, Debug)]
 pub struct Window {
-    /// The cursor options of this window. Cursor icons are set with the `Cursor` component on the
+    /// The cursor options of this window. Cursor icons are set with the `CursorIcon` component on the
```

Comment:
👍  ; I think that's important to note to ease with migration, that being said this property actually refers to `bevy_render`, and it's annoying that we can't link to it through a real link :( (also, should we at all ? 🤔)

Reply:
no worries! we have had a whole conversation about it inside a discord thread

tl;dr ideally this would live inside of `bevy_winit`, to keep ergonomics the same since most of the use cases regarding icons right now involve `Query<&Window>` and reading the `cursor_options` field which is of the `CursorOptions` type, which originally had `CursorIcon`. Now that we have custom cursor support, this had to live inside `bevy_render` because the custom cursors require access to `Image` which can't be imported inside of `bevy_winit` nor `bevy_window` due to circular dependencies.

though I do agree that the comment should be linked properly - I tried adding the fully qualified name on another comment, but I wasn't sure about how to run the docs to check if it worked locally (i'm still a rust toolchain noob)

Reply:
thanks for the insight ! 

> I tried adding the fully qualified name on another comment

Nice try! the CI docs step fails due to that. Check it out (and I recommend looking at the .github workflow file to see more details on what commands are run :+1:

I fear it's impossible to link because bevy_window doesn't rely on bevy_render :(

well I imagine we theoretically could by adding the dependency if building docs, but I personally wouldn't approve that :/

<details><summary>Continued rambling about bringing CursorIcon back to `bevy_window`</summary>
<p>

I'd prefer a `Custom` empty enum variant, and let the other crates handle their custom thing however they want 🤔.

Or a generic Custom<T> ? or a Box with a dyn trait something inside ? 

</p>
</details> 

> I wasn't sure about how to run the docs

`cargo doc --open` will bring you far enough 🤞